src/TestUtils/src/UITest.Appium/Actions/AppiumAndroidThemeChangeAction.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Diagnostics;
+using UITest.Core;
+
+namespace UITest.Appium
+{
+	public class AppiumAndroidThemeChangeAction : ICommandExecutionGroup
+	{
+		const string SetLightTheme = "setLightTheme";
+		const string SetDarkTheme = "setDarkTheme";
+
+		readonly List<string> _commands = new()
+		{
+			SetLightTheme,
+			SetDarkTheme
+		};
+
+		protected readonly AppiumApp _app;
+
+		public AppiumAndroidThemeChangeAction(AppiumApp app)
+		{
+			_app = app;
+		}
+
+		public CommandResponse Execute(string commandName, IDictionary<string, object> parameters)
+		{
+			if (commandName == SetLightTheme)
```

Comment:
Sorry @kubaflo but in the used Appium version (rc7), this option is not available. Could you restore the previous `AppiumAndroidThemeChangeAction ` version? 

`System.NotImplementedException : Unknown mobile command "setUiMode". Only shell,execEmuConsoleCommand,dragGesture,flingGesture,doubleClickGesture,clickGesture,longClickGesture,pinchCloseGesture,pinchOpenGesture,swipeGesture,scrollGesture,scrollBackTo,scroll,viewportScreenshot,viewportRect,deepLink,startLogsBroadcast,stopLogsBroadcast,acceptAlert,dismissAlert,batteryInfo,deviceInfo,getDeviceTime,changePermissions,getPermissions,performEditorAction,startScreenStreaming,stopScreenStreaming,getNotifications,openNotifications,listSms,type,replaceElementValue,pushFile,pullFile,pullFolder,deleteFile,isAppInstalled,queryAppState,activateApp,removeApp,terminateApp,installApp,clearApp,backgroundApp,getCurrentActivity,getCurrentPackage,startActivity,startService,stopService,broadcast,getContexts,getAppStrings,installMultipleApks,lock,unlock,isLocked,refreshGpsCache,startMediaProjectionRecording,isMediaProjectionRecordingRunning,stopMediaProjectionRecording,getConnectivity,setConnectivity,toggleGps,isGpsEnables,hideKeyboard,isKeyboardShown,pressKey,getDisplayDensity,getSystemBars,fingerprint,sendSms,gsmCall,gsmSignal,gsmVoice,powerAc,powerCapacity,networkSpeed,sensorSet,getPerformanceData,getPerformanceDataTypes,statusBar,screenshots commands are supported.`